components/layout_2020/display_list/stacking_context.rs

```diff
@@ -833,6 +857,14 @@ pub(crate) enum StackingContextBuildMode {
 }
 
 impl Fragment {
+    #[cfg_attr(
+        feature = "tracing",
+        tracing::instrument(
+            name = "display_list::build_stacking_context_tree",
+            skip_all,
+            fields(servo_profiling = true)
+        )
+    )]
```

Comment:
can we avoid tracing the recursion of this function?

Reply:
yeah, good point. It was flooding the logs. I have updated the code. Now we are capturing the span context only for the current call of `build_stacking_context_tree`

![image](https://github.com/user-attachments/assets/962c00c6-70fc-42b7-a85c-6a7ad7525b20)

Reply:
thanks! there’s an event with a similar name in a match arm for Fragment::AbsoluteOrFixedPositioned(fragment), do we still need that?

Reply:
Idea was to measure the event inside match but now that thinking about it again, it doesn't make much sense to have that only for `AbsoluteOrFixedPositioned`. I have removed this and a few more events in [880b7e5](https://github.com/servo/servo/pull/34128/commits/880b7e5f6018a6d0d1e108994c72ffd3ac733c37),